ls1intum/Artemis
# Bugfix/import programming

### Checklist

### Motivation and Context

Closes #3328.

### Description

When the exercise id was `undefined`, the instructions component would try to load `/api/exercises/undefined/hints` and since `undefined` is not a number, an exception would be thrown: `Failed to convert value of type 'java.lang.String' to required type 'java.lang.Long'; nested exception is java.lang.NumberFormatException: For input string: "undefined"` (which causes the "400 Bad Request")

Also fixes the title for importing.
Before:
![grafik](https://user-images.githubusercontent.com/72132281/116741649-ad54d900-a9f6-11eb-9dad-6723489b6c2d.png)
After:
![grafik](https://user-images.githubusercontent.com/72132281/116741660-b0e86000-a9f6-11eb-9c11-9be6afd615c0.png)

### Steps for Testing

1. Find or create a programming exercise.
2. Find or create an exam.
3. Import the programming exercise to the exam. Make sure points, title, instructions, etc. are set correctly when starting the import.
4. Repeat the same for importing an exercise to a course.

Example for how it might look:
![grafik](https://user-images.githubusercontent.com/72132281/116741933-f3aa3800-a9f6-11eb-866c-5b110f744614.png)
